admin_guide/managing_networking.adoc

```diff
@@ -898,8 +898,52 @@ using the *ovs-networkpolicy* plug-in.
 In a cluster
 xref:../install_config/configuring_sdn.adoc#install-config-configuring-sdn[configured
 to use the *ovs-networkpolicy* plug-in], network isolation is controlled
-entirely by `NetworkPolicy` objects and the link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/network-policy.md[associated Namespace annotation]. In particular, by default, all projects are able to access pods
-in all other projects. The project to be isolated must first be configured to
-opt in to isolation by setting the proper annotation on its Namespace object,
-and then creating `NetworkPolicy` objects indicating the incoming connections to
-be allowed.
+entirely by
+link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/network-policy.md[`NetworkPolicy`
+objects]. In particular, by default, all projects are able to access pods in all
+other projects. The project to be isolated must first be configured to opt in to
```

Comment:
1. Should Namespace object be `Namespace`?

2. Lets change this to active, something like: 

> To isolate a project, opt into isolation by configuring the `Namespace` object, and then create a `NetworkPolicy` object to indicate the allowed incoming connections.